### PR TITLE
nudges north an south hunter dome up by 1 pixel to match the rest

### PR DESCRIPTION
**Pull request by kcin2001**
it bothers me ok, and im pretty sure i fixed it once before

### CODE REVIEW

**Comment by kcin2001** [comment]
ugh not ready, my ammend didnt work but i wasnt sure

**Comment by kcin2001** [comment]
ok its all ready, tortoise decided to be weird about it

File: code/modules/mob/new_player/sprite_accessories_vr.dm [hunk 1]
```diff
@@ -458,10 +458,10 @@
 		species_allowed = list(SPECIES_XENOHYBRID)
 		gender = NEUTER
 
-	xeno_head_hunter_color
+	xeno_head_hunter_color //editing code instead of the sprite because its easier to track
 		name = "Hunter dome"
-		icon = 'icons/mob/human_face_vr.dmi'
-		icon_add = 'icons/mob/human_face_vr_add.dmi'
+		icon = 'icons/mob/human_face_ch.dmi'
+		icon_add = 'icons/mob/human_face_ch_add.dmi'
```
Comment (TheFurryFeline) [inline code/modules/mob/new_player/sprite_accessories_vr.dm:464]: //Chompstation edit comment.

**Comment by TheFurryFeline** [comment]
Also, update your master branch. I'm not seeing Travis running presumably because your files are outdated.

File: code/modules/mob/new_player/sprite_accessories_vr.dm (hunk 1 above)
Comment (kcin2001) [reply code/modules/mob/new_player/sprite_accessories_vr.dm:464]: i dont know why i didnt say chompstation edit in the comment, smoothbrain

**Comment by kcin2001** [comment]
if it was because of my master branch being out of date the last pr would have the same issue

**Pull request merged**